qtp_biom/summary.py

```diff
@@ -29,8 +30,14 @@
 
 def _generate_html_summary(biom_fp, metadata, out_dir, is_analysis, tree=None):
     if is_analysis:
-        metadata = qiime2.Metadata(pd.DataFrame.from_dict(
-            metadata, orient='index'))
+        # we need to save and load the df so qiime does it's magic for parsing
+        # columns
+        fd, path = mkstemp()
+        df = pd.DataFrame.from_dict(metadata, orient='index')
+        df.to_csv(path, index_label='#SampleID', na_rep='', sep='\t',
```

Comment:
I am in error. to_csv() takes a path to a file, not a path to a directory; these changes should be fine.